.env.example

```diff
@@ -0,0 +1,4 @@
+PROVIDER_EMAIL=you@your-domain.com
+PROVIDER_PASSWORD=your-omnistrate-password
```

Comment:
Are you sure that we need this? -- I recollect @rushil95 mentioning that this is no longer needed. Can you please talk to him?

Reply:
For local development I think we still need it, at least last week I had to valorize it.
We could always rename it to `.env.local` if this is the case

Reply:
The customers don't use this environment variable. We use it internally for deploying to fly as we need to use the dev server for those deployments. 
@mbianchidev While running locally you could just set the baseDomain to https://api.omnistrate.dev in the above file